.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+
+env:
+  - DEP_VERSION="0.5.3"
+
+before_install:
+  - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
+  - chmod +x $GOPATH/bin/dep
+
+install:
+  - dep ensure
```

Comment:
[This doc](https://docs.travis-ci.com/user/languages/go/#godep-support) suggests that go dep takes care of itself if you don't specify an install step. Did that not work for some reason?

Reply:
took me a sec to figure this one out... `godep` is not the same thing as `dep`, and neither of them are the officially supported tool as of go11. https://github.com/golang/go/wiki/PackageManagementTools

tl;dr; we need this to use dep, but probably should switch to the latest hotness at some point.